tests/scripts/local_monitoring_tests/Local_monitoring_tests.md

```diff
@@ -164,3 +164,19 @@ Local monitoring tests can also be run without using the test_autotune.sh. To do
 
 Note: You can check the report.html for the results as it provides better readability
 
+
+### Accelerator Test:
+
+Kruize 0.1 supports the Accelerator Recommendations which provide right sized MIG config as recommendations.
+
+The test `test_list_recommendations.py::test_accelerator_recommendation_if_exists` is created to check if the accelerator recommendations are in expected format.
+
+#### Prerequisites to run the test:
+
+In addition to the pre-requisites mentioned above we need to make sure that a workload with name `human-eval-benchmark` is running in the namespace `unpartitioned` and has the accelerator usage data.
```

Comment:
Please provide steps on how to run this workload or link to a doc that has the steps

Reply:
It's one of our benchmarks `human-eval-benchmark` can I add link to it?

Reply:
Added a link of human eval benchmark in the doc.